src/filter.ts

```diff
@@ -32,11 +37,24 @@ export interface RangeFilter {
    * @maxItems 2
    * @minItems 2
    */
-  range: any;
+  range?: any;
+  lt?: number;
```

Comment:
Can you help copy description from the docs to here.  (Altair folks uses description in the JSON schema as part of their iPython autocomplete as well.)